src/gui/tray/NotificationCache.cpp

```diff
@@ -0,0 +1,28 @@
+#include "NotificationCache.h"
+#include <qhash.h>
+
+namespace OCC {
+
+NotificationCache::NotificationCache(QObject *parent)
+    : QObject(parent) {};
```

Comment:
This semicolon shouldn't be here. Also each curly brace should be on its own line.